addons/l10n_in_edi/models/account_move.py

```diff
@@ -225,7 +229,32 @@ def _l10n_in_edi_send_invoice(self):
                         "doc_date": self.invoice_date and self.invoice_date.strftime("%d/%m/%Y"),
                     }
                 )
-                if not response.get("error"):
+                mismatch_error = []
+                if jwt:
+                    try:
+                        if SignedInvoice := response.get("data", {}).get('SignedInvoice', {}):
```

Comment:
```suggestion
                        if SignedInvoice := response.get('data', {}).get('SignedInvoice', {}):
```
Please also use single quotes for the other ones